### PR TITLE
UI Test refactoring

### PR DESCRIPTION
**Pull request by audax**
Because running setup code in a constructor is not reliable.

### CODE REVIEW

**Comment by audax** [comment]
retest this please

**Comment by audax** [comment]
I just don't see why suddenly the BoxProvider tests start to fail.

**Comment by julianseeger** [comment]
please merge/rebase

File: qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/ImportExportContactsUITest.java [hunk 1]
```diff
@@ -233,16 +222,16 @@ public void testExportManyContact() throws JSONException {
         assertNotNull(file1);
         goToContacts();
         openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
-
         Spoon.screenshot(mActivity, "exportAll");
-		pressBack();
 		Intent data = new Intent();
 		data.setData(Uri.fromFile(file1));
-		ContactFragment contactFragment = (ContactFragment) mActivity.getFragmentManager().findFragmentById(R.id.fragment_container);
+		ContactFragment contactFragment =
```
Comment (julianseeger) [inline qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/ImportExportContactsUITest.java]: Indentation looks weird, removing tabs and breaking after a dot may increase beauty here ^^

**Comment by julianseeger** [comment]
lets imagine the build was green

**Pull request merged**